src/components/PlaceBet.jsx

```diff
@@ -133,9 +134,11 @@ class PlaceBet extends React.Component {
       });
     }, () => {
       const classType = 'danger';
-      const message = 'No account connected, '
-        + 'connect with a Web3-compatible wallet like MetaMask';
-      const alertDict = { classType, message };
+      const noAccountConnected = () => (<>
+        {'No account connected, connect with a Web3-compatible wallet like '}
+        <MetaMaskLink />
+      </>)
```

Comment:
what are these empty tags for? Is it like a `div` shortcut? Will the linter be OK about it?

Reply:
They are React Fragments (https://reactjs.org/docs/fragments.html) since React can't render two components at once, it needs to wrap the components in something before we were using an empty div which only adds unnecessary DOM nodes.

Reply:
Nice I was always there, thanks for the link!